README.md

```diff
@@ -1,20 +1,73 @@
+[![GitHub CI](https://img.shields.io/github/workflow/status/Sage-Bionetworks/mongo/ci.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&logo=github)](https://github.com/Sage-Bionetworks/mongo/actions)
+[![GitHub License](https://img.shields.io/github/license/Sage-Bionetworks/mongo.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&logo=github)](https://github.com/Sage-Bionetworks/mongo/blob/main/LICENSE)
+[![Docker Pulls](https://img.shields.io/docker/pulls/sagebionetworks/mongo.svg?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&label=pulls&logo=docker)](https://hub.docker.com/r/sagebionetworks/mongo)
 
 Hardened MongoDB Docker image
 
 ## Specification
 
-For better security, the entrypoint script is modified to provide an alternative
-to Mongo's default database and user.
+This image extends the official [mongo image](https://hub.docker.com/_/mongo)
+to provide the features described below.
 
-This container
+### Create non-root user at startup
 
-- creates a database named according to the environment variable
-  `${MONGO_INITDB_DATABASE}`
-- create a DB user at startup using the values of the environment variables
-  `${MONGO_USERNAME}` and `${MONGO_PASSWORD}`
+By default, a user connects to the DB using the root account specified by
+the credentials defined by the following official environment variables
+
+- `MONGO_INITDB_ROOT_USERNAME`
+- `MONGO_INITDB_ROOT_PASSWORD`
+
+as shown in this official example:
+
+```console
+docker run -d --network some-network --name some-mongo \
+    -e MONGO_INITDB_ROOT_USERNAME=mongoadmin \
+    -e MONGO_INITDB_ROOT_PASSWORD=secret \
+    mongo
+```
+
+**Issue:** This root account has elevated priviledges that allow it to perform any actions
+in the DB. In practice, you may prefer to restrict the priviledges that your
+application has access to for better security.
+
+**Solution:** When a container is started for the first time it will execute the
+script [add-db-user](add-db-user) that creates a user account with priviledges
+limited to the role [dbAdmin]. This role does not grant privileges for user
+and role management. Moreover, this user access is limited to the use of a
+single database specified by the official environment variable
+`MONGO_INITDB_DATABASE`. This user account is created only if the environment
+variables listed below are specified:
+
+- `MONGO_USERNAME`
+- `MONGO_PASSWORD`
+
+Example:
+
+```console
+docker run -d --network some-network --name some-mongo \
+    -e MONGO_INITDB_ROOT_USERNAME=mongoadmin \
+    -e MONGO_INITDB_ROOT_PASSWORD=secret \
+    -e MONGO_INITDB_DATABASE=appdb \
+    -e MONGO_USERNAME=appuser \
+    -e MONGO_PASSWORD=secret \
+    sagebionetworks/mongo
+```
+
+Note the use of the image [sagebionetworks/mongo].
```

Comment:
I'm not quite sure what this is supposed to elucidate? What is the significance of highlighting the usage of the `sagebionetworks/mongo` image?

Reply:
The reason was because I used to official `mongo` image in the first example and `sagebionetworks/mongo` in the second example. I updated the README to use the image `sagebionetworks/mongo` in all examples.